spec/api-native-image-spec.js

```diff
@@ -25,6 +25,10 @@ describe('nativeImage module', () => {
 
       const imageC = nativeImage.createFromBuffer(imageA.toJPEG(100))
       assert.deepEqual(imageC.getSize(), {width: 538, height: 190})
+
+      const imageD = nativeImage.createFromBuffer(imageA.toBitmap(),
+        {width: 538, height: 190})
```

Comment:
I added some tests for bitmaps and the other types. PNG and JPEG is not affected by width/height, because the size can be decoded from the buffer, but bitmaps don't have that so they always have the same size, that is specified. That said they won't crash when a wrong size is passed, just the picture will be messed up.